wasmCloud/wasmcloud-otp
# Added actors started and actors stopped events

## Feature or Problem
This PR adds three new events to the ActorSupervisor: `actors_started`, `actors_start_failed`, and `actors_stopped`. These events are published once after control interface requests to start/stop actors, once they finish, along with the total count started/stopped.

## Related Issues
No issue created, but this will allow `wadm` to be a lot smarter in terms of knowing when a request to start/stop an actor actually completes.

## Release Information
`v0.63.0`

## Consumer Impact
These three events will be added, so consumers of the event stream will need to consider these if they subscribe to all events. As I'm writing this, I realize that I need to update the wasmCloud dashboard to ignore these.

## Testing



Built on platform(s)


Tested on platform(s)

### Unit Test(s)


### Acceptance or Integration


### Manual Verification
Tested to ensure these events publish properly and contain the proper information.
